R/language.R

```diff
@@ -33,10 +49,16 @@
 #' is_atomic_lang(quote(call()))
 #'
 #' # With the exception of names, atomic language objects have the
-#' # property of being self-quoting:
+#' # property of being self-quoting literals:
```

Comment:
hmm yes it would make sense from the perspective of computing on the language: symbols need to be inspected in a way that literals don't.